schema.sql

```diff
@@ -44,3 +44,32 @@ ALTER TABLE animals ADD COLUMN species_id INT, ADD COLUMN owner_id INT;
 ALTER TABLE animals ADD CONSTRAINT fk_animals_species FOREIGN KEY (species_id) REFERENCES species (id);
 
 ALTER TABLE animals ADD CONSTRAINT fk_animals_owners FOREIGN KEY (owner_id) REFERENCES owners (id);
+
+-- add join table
+
+-- Create a table named vets with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+-- age: integer
+-- date_of_graduation: date
+CREATE TABLE vets (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR,
```

Comment:
- [ OPTIONAL] 
Consider specifying the maximum length of the name column. Below are some advantages of doing that.

- Data Integrity: By setting a maximum length, you can enforce data integrity and prevent data truncation. If a user attempts to insert or update a value that exceeds the specified length, the database system will raise an error, ensuring that the data remains within the defined boundaries.

- Storage Optimization: Specifying an appropriate maximum length can help optimize storage space. If you know that the data in a column will not exceed a certain length, you can set a smaller maximum length, which can reduce the overall storage requirements for the database.

- Performance Improvement: Setting a specific maximum length can contribute to better query performance. When the database knows the maximum length of a field, it can allocate the appropriate amount of storage and optimize query execution plans accordingly, leading to faster data retrieval and manipulation.

- Consistency and Standardization: By specifying a maximum length, you establish a consistent format for the field across the database. This promotes standardization, making it easier to understand and work with the data. It also helps ensure that all records adhere to the same length constraint.

- Application Development: Specifying a maximum length can be beneficial during application development. It provides developers with clear guidelines on the expected size of the data, allowing them to design user interfaces, input validation, and business logic accordingly. It can also prevent issues related to rendering or displaying data that is too long for a given layout or screen